File: app/src/main/java/plant/testtree/camerademo/activity/gallary/CursorHandler.java
package plant.testtree.camerademo.activity.gallary;

import android.database.Cursor;

/* loaded from: classes.dex */
public interface CursorHandler<T> {

    /* loaded from: classes.dex */
    public static final class CC {
        public static String[] getProjection() {
            return new String[0];
        }
    }

    T handle(Cursor cursor);
}

File: app/src/main/java/com/iphonecamera/allinone/cameraediting/activity/gallary/CursorHandler.kt
package com.iphonecamera.allinone.cameraediting.activity.gallary

import android.database.Cursor

interface CursorHandler<T> {
    object CC {
        val projection: Array<String?>
            get() = arrayOfNulls(0)
    }

    fun handle(cursor: Cursor?): T
}